assets/src/js/header.js

```diff
@@ -89,6 +89,7 @@ const closeInactiveNavElements = event => {
 
   activeElements.forEach(button => {
     const buttonTarget = button.dataset && button.dataset.bsTarget;
+    // eslint-disable-next-line @wordpress/no-unused-vars-before-return
```

Comment:
Instead of disabling this here, we could move the `buttonTargetElement` declaration after the `return` statement